### DISCUSSION TITLE
q-input gives Typescript error regarding use of "autocomplete" prop?

### DISCUSSION THREAD

**Discussion by jimmiejackson414** [quasarframework/quasar#13643] [General - Components / Directives / etc] [answered]
I'm using Quasar v2.7.1, and noticed that my <q-input /> elements on my Login form are emitting a TS error. I recently had upgraded to 2.71, so I'm not sure if that's the cause?

```
<q-input
  ref="emailRef"
  v-model="email"
  :disable="isSubmitting"
  outlined
  dense
  label="Email"
  type="email"
  autocomplete="email" // <-- TS Error
  :rules="emailRules">
  <template #prepend>
    <q-icon name="bi-envelope" />
  </template>
</q-input>
```

The error: 
```
Property 'autocomplete' does not exist on type 'IntrinsicAttributes & VNodeProps & AllowedComponentProps & ComponentCustomProps & QInputProps'.
```

Looking at the [docs](https://quasar.dev/vue-components/input#native-attributes), they mention that any attributes not part of the API will be passed down to the native element, and `autocomplete` is one of those examples.

What might be causing this?

**Comment by suleimanas** [comment]
Do you still have vetur installed, if so try to disable it.
i found it to be the issue in my case

**Reply by jimmiejackson414** [reply]
I don't, I'm using Volar.

**Comment by yusufkandemir** [comment, marked as answer]
It's happening because of recent changes in Volar. See #13602 for more info, references, and the solution. If it doesn't solve your problem, please continue the discussion on that issue instead of here.

**Reply by jimmiejackson414** [reply]
Interesting, thank you!
